src/Shell/Task/ModelTask.php

```diff
@@ -691,14 +691,14 @@ public function getValidation($model, $associations = [])
      * @param \Cake\Database\Schema\TableSchema $schema The table schema for the current field.
      * @param string $fieldName Name of field to be validated.
      * @param array $metaData metadata for field
-     * @param string $primaryKey The primary key field
+     * @param array $primaryKey The primary key field
```

Comment:
Cant primary key also be a string?
Maybe we should do string|array and cast inside the in_array() call?

Reply:
I see the cast is happening before calling this method: `$primaryKey = (array)$schema->primaryKey();`
Then you can just add the array typehint in the signature IMO.

Reply:
Done.